src/NuGet.Clients/NuGet.VisualStudio.Common/IDE/VsMonitorSelectionExtensions.cs

```diff
@@ -34,10 +35,28 @@ public static EnvDTE.Project GetActiveProject(this IVsMonitorSelection vsMonitor
                 // multiple items are selected.
                 if (pitemid == (uint)VSConstants.VSITEMID.Selection)
                 {
-                    return null;
+                    uint numberOfSelectedItems;
+                    int isSingleHierarchyInt;
+                    if (ErrorHandler.Succeeded(ppMIS.GetSelectionInfo(out numberOfSelectedItems, out isSingleHierarchyInt)))
```

Comment:
There are two calls to `GetSelectionInfo` in this PR, with similar surrounding code. Maybe an opportunity to pull out a helper.